cmd/geth/logtestcmd_active.go

```diff
@@ -51,9 +50,6 @@ func (c customQuotedStringer) String() string {
 // logTest is an entry point which spits out some logs. This is used by testing
 // to verify expected outputs
 func logTest(ctx *cli.Context) error {
-	// clear field padding map
-	debug.ResetLogging()
```

Comment:
We had to add this to have the log tests deterministic. Curious if it works without it

Reply:
That's why I was asking how this thing is used? The only was I saw was running `geth logtest`, in which case there's nothing logging that would break it.

Reply:
There's no documentation on how to use this command, so I have no idea if I broke something or not.

Reply:
The tests will tell you if you did

Reply:
```
 go test ./cmd/geth -tags integrationtests -run TestLogging
```

Reply:
CI seems happy about it though :)